src/ROUTES.ts

```diff
@@ -795,13 +795,10 @@ const ROUTES = {
         route: 'settings/workspaces/:policyID/accounting/xero/import/tracking-categories',
         getRoute: (policyID: string) => `settings/workspaces/${policyID}/accounting/xero/import/tracking-categories` as const,
     },
-    POLICY_ACCOUNTING_XERO_TRACKING_CATEGORIES_MAP_COST_CENTERS: {
-        route: 'settings/workspaces/:policyID/accounting/xero/import/tracking-categories/cost-centers',
-        getRoute: (policyID: string) => `settings/workspaces/${policyID}/accounting/xero/import/tracking-categories/cost-centers` as const,
-    },
-    POLICY_ACCOUNTING_XERO_TRACKING_CATEGORIES_MAP_REGION: {
-        route: 'settings/workspaces/:policyID/accounting/xero/import/tracking-categories/region',
-        getRoute: (policyID: string) => `settings/workspaces/${policyID}/accounting/xero/import/tracking-categories/region` as const,
+    POLICY_ACCOUNTING_XERO_TRACKING_CATEGORIES_MAP: {
+        route: 'settings/workspaces/:policyID/accounting/xero/import/tracking-categories/mapping/:categoryId/:categoryName',
+        getRoute: (policyID: string, categoryId: string, categoryName: string) =>
+            `settings/workspaces/${policyID}/accounting/xero/import/tracking-categories/mapping/${categoryId}/${categoryName}` as const,
```

Comment:
We had similar issue where we put `categoryName` in the path (https://github.com/Expensify/App/issues/37885).
I think we should encode the category name here.
```suggestion
            `settings/workspaces/${policyID}/accounting/xero/import/tracking-categories/mapping/${categoryId}/${encodeURIComponent(categoryName)}` as const,
```

You can test this case by creating a new category on Xero with `/` in it. For example:
<img width="677" alt="Screenshot 2024-05-27 at 13 46 02" src="https://github.com/Expensify/App/assets/16502320/cae1e7fc-ebe6-41c8-806d-f97214aa18f8">

Reply:
Nice catch! 

I've fixed it:
![image](https://github.com/Expensify/App/assets/21549771/7b0c709c-7dca-4a62-8e2a-c2d68410d464)